packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/intel/IpGeolocateData.java

```diff
@@ -0,0 +1,49 @@
+package cloud.pangeacyber.pangea.intel;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class IpGeolocateData {
```

Comment:
Would this be analogous to a dataclass in python? 

also completely other random question. do we want to have helpers that pretty print this data or something like that?

Reply:
Yes, sort of I think. How would you do it?
Totally agree about the pretty print, I've thought about it to implement in API errors, but I didn't do it yet

Reply:
we can file a ticket for this down the line or if a customer asks for it